server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/LocalStorageManager.scala

```diff
@@ -460,7 +466,11 @@ private[worker] final class LocalStorageManager(
     var exception: IOException = null
     val suggestedMountPoint = location.getStorageHint.getMountPoint
     while (retryCount < RssConf.createFileWriterRetryCount(conf)) {
-      val dirs = workingDirsSnapshot(suggestedMountPoint)
+      val dirs = getUsableWorkingDirs(suggestedMountPoint)
+      if (dirs.isEmpty) {
+        logDebug("All disks are not available.")
```

Comment:
I'll add logic here to flush data to HDFS. This is intended to keep the main branch running if all disk space is low.